<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>411</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>502</width>
    <height>411</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>502</width>
    <height>411</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View/Edit/Delete</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.needed/icons/alarm.png</normaloff>.needed/icons/alarm.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lwAlarmList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="teAlarmDetails">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>481</width>
      <height>251</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewAlarmInfo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>.needed/icons/view.png</normaloff>.needed/icons/view.png</iconset>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditAlarm">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>.needed/icons/edit.png</normaloff>.needed/icons/edit.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteAlarm">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>.needed/icons/delete.png</normaloff>.needed/icons/delete.png</iconset>
    </property>
    <property name="shortcut">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWhoAmI">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>#Who_Am_I_?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
